browser/resources/settings/default_brave_shields_page/components/brave_adblock_editor.html

```diff
@@ -0,0 +1,108 @@
+<style include="cr-shared-style settings-shared iron-flex">
```

Comment:
Small bug on light theme perhaps where the active line color looks a bit less subtle than on dark themes:
![image](https://user-images.githubusercontent.com/741836/169169703-c51f0d26-5d7c-407b-a3a4-63582beacc76.png)

Reply:
removed line highlighting feature